src/pytest_html/scripts/main.js

```diff
@@ -91,8 +91,15 @@ const renderDerived = (tests, collectedItems, isFinished) => {
         const accTime = tests.reduce((prev, { duration }) => prev + duration, 0)
         const formattedAccTime = formatDuration(accTime)
         const testWord = numberOfTests > 1 ? 'tests' : 'test'
-        const innerText = `${numberOfTests} ${testWord} ran in ${formattedAccTime} seconds.`
-        document.querySelector('.run-count').innerText = innerText
+
+        let durationText
+        if (formattedAccTime.hasOwnProperty('ms')) {
+          durationText = formattedAccTime.ms
+        } else {
+          durationText = formattedAccTime.seconds
+        }
```

Comment:
```suggestion
        const durationText = formattedAccTime.ms ?? formattedAccTime.seconds
```

Reply:
I went with the ternary option, the `??` is super nice but I feel it's too much magic for your average pythonista to handle. 😅  @drRedflint